src/client/components/pages/Courses/tableFields.tsx

```diff
@@ -612,7 +612,7 @@ export const tableFields: CourseInstanceListColumn[] = [
     }: FieldContentProps): ReactElement => {
       const { notes } = course;
       const hasNotes = notes && notes.trim().length > 0;
-      const titleText = hasNotes ? 'View/Edit Notes' : 'Add Notes';
+      const titleText = `${!hasNotes ? 'Add' : 'View/Edit'} notes for ${course.catalogNumber}`;
```

Comment:
Now that we're limiting who can actually edit notes, the "Add" and "Edit" text doesn't quite feel right. What if we just simplify it something like "Open Notes for ${course.catalogNumber}"? Otherwise trying to cover all the conditional cases feels like it would get messy very quickly...

Reply:
Good point. Fixed in bde7402350b4de17a293a43a4258889221a85763